### PR TITLE
Feature/countdown

### LINKED ISSUE #20: 1.4 countdown in tests
All funcionality for a time restriction when filling out test

### PR DESCRIPTION
**Pull request by Kinga-Zawarczynska**
Add countdown with basic logic - if the time is up, the test will end with information of the reason
Fix console warning in pagination component

Countdown is written with H1 tag - for a moment let's not worry about that, it'll be fixed as soon as layout of the tests will be clear

### CODE REVIEW

File: src/components/Countdown/Countdown.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    useEffect(() => {
+        if (!timeLeft) return;
+
+        const intervalId = setInterval(() => {
+            setTimeLeft(timeLeft - 1);
+        }, 1000);
+        return () => clearInterval(intervalId);
+    }, [timeLeft]);
+
```
Comment (wszczawinski) [inline src/components/Countdown/Countdown.js:14]: refactor useEffect content into separate function

File: src/components/Pagination/Pagination.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ export const Pagination = ({ postsPerPage, totalPosts, paginate, currentPage, el
               </Button>
             </div>
           ))}
-        </div>
+        </section>
       </nav>
     );
   }
```
Comment (wszczawinski) [inline src/components/Pagination/Pagination.js:34]: Semicolons (in imports) - one convention, is there tabIndex prop in Button component?

File: src/components/TestQuestion/TestQuestion.js [hunk 3]
```diff
@@ -13,13 +26,19 @@ export function TestQuestion(props) {
...
+                <Countdown seconds={testTimeLeft} setTestTimeLeft={setTestTimeLeft}/>
                <div className={styles.question}>{question}</div>
-               <div className={styles.answears}>
+               <div className={styles.answers}>
                    <ul>
-                       <li onClick={() => calculatePoints(currentPage, answear1)}>{answear1.value}</li>
+                       <li className={liClassName} onClick={() => calculatePoints(currentPage, answear1)} >{answear1.value}</li>
                        <li onClick={() => calculatePoints(currentPage, answear2)}>{answear2.value}</li>
                        <li onClick={() => calculatePoints(currentPage, answear3)}>{answear3.value}</li>
                    </ul>
```
Comment (wszczawinski) [inline src/components/TestQuestion/TestQuestion.js:44]: Semicolons - one convention.

**Review by wszczawinski — APPROVED** [review]
Looks good for now, but needs to be finished (organized and refactored)  in next sprint.

**Review by MateuszRostkowski — APPROVED** [review]

**Pull request closed without merging**